polymer/index.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+
+    <title>abe-polymer2</title>
+    <meta name="description" content="test of Polymer">
```

Comment:
Update the `content`. https://github.com/PolymerElements/polymer-starter-kit/blob/e96167dac682c847008aee8ed40f2040b361da8f/index.html#L33-L60 has some good defaults for this.

Reply:
Added the description.